### ISSUE TITLE
Octicons scale fail

### ISSUE THREAD

**Issue by 237dmitry** [ryanoasis/nerd-fonts#1288]
[#1240](https://github.com/ryanoasis/nerd-fonts/issues/1240)
[#1261](https://github.com/ryanoasis/nerd-fonts/pull/1261)

Nothing was fixed in 3.0.2. The same issue as in the initial one. The problem is not in circles but in `\UE0B4`

3.0.0:
![300](https://github.com/ryanoasis/nerd-fonts/assets/78153320/4fda4447-e176-4579-9665-8e2e2ecf7f5a)

3.0.1 & 3.0.2:
![302](https://github.com/ryanoasis/nerd-fonts/assets/78153320/a10b3bc7-f224-4ea7-a321-cf5415911ff2)

**Comment by Finii** [comment]
![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/48ec99db-62cf-49b8-9f3b-8b5ef91b2312)

From #1240, I asked you if `ea71` is the problematic glyph and you said it is.
But in the top of that issue (I just noticed that) you use `f444`. That is a completely different set, and I did not look at that, of course.

So please use `ea71`, THAT is fixed.

**Comment by Finii** [comment]
Octicons is shifted from `F000` to `F400`.
So `F444` is `F044` in the icon font.

Why is that in `triangles`?

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/1e7d2643-445c-463e-ae57-9566d4db47cc)

**Comment by Finii** [comment]
On the update we changed to `plus`

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/8d06b0a4-b130-40d3-a524-ec7282a724bd)

```
CommitDate: Wed May 3 09:58:23 2023 +0200

    font-patcher: Fix some Octicons scales

    [why]
    With the Octicons update some codepoint contents changed.
    Also some symbols scale still wrong.

    [how]
    Check fonts and add / drop entries.
    Use GlyphsToScale+ to correct position of Github text icon.

    Fixes: #1198
```

**Comment by Finii** [comment]
> The problem is not in circles but in \UE0B4

Repeating this over and over will not help (and is wrong, as I tried to explain before).
